travis/clone_oca_dependencies

```diff
@@ -39,8 +40,10 @@ travis_repo_slug = os.environ.get(
 travis_repo_owner = travis_repo_slug.split("/")[0]
 
 
-def parse_depfile(depfile):
+def parse_depfile(depfile, default_owner):
     deps = []
+    url_exp = (r'^(?P<proto>git@|http(s)?:\/\/)(?P<domain>[a-z\.\-]+)(:|\/)'
```

Comment:
You can use `git_update.get_repo_data` to get host, owner and repo fields from github url